packages/jss-observable/src/index.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+import isObservable from 'is-observable'
+import type StyleRule from 'jss/src/rules/StyleRule'
+import createRule from 'jss/lib/utils/createRule'
+import type {Rule, RuleOptions, JssStyle} from '../../jss/src/types'
```

Comment:
Is that ok to import types that way? I thought they have to be part of public interface and imported differently

Reply:
Well, the types are going to be stripped out anyway. It's not the best solution but exporting everything isn't either.

Reply:
Well, we could evtl go with this for now and rethink later